modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -0,0 +1,84 @@
+. Synchronize images to the terminals
+
+Each procedure is detailed in this section.
+
+If you have many terminals, and would prefer this to be handled with a script, see xref:retail:retail-mass-config[].
+
+Before terminals can be deployed, ensure you have prepared a Saltboot-based operating system image.
+For how to build OS images, see xref:administration:image-management.adoc[].
+
+[IMPORTANT]
+====
+After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminal has connected successfully to the branch server, and you have not connected directly to the {susemgr} server by mistake.
+====
+
+
+== Create a Hardware Type group
+
+Each terminal requires a specific hardware type, which contains information about the product name and terminal manufacturer.
+However, the {susemgr} database does not have this information to begin with.
+In order to tell {susemgr} what image to deploy on each terminal, you can set hardware type groups.
+After you have created your new hardware type group, you can apply the [systemitem]``saltboot-formula`` to the group and configure it for your environment.
+
+Hardware types allow you to group devices according to manufacturer and device name, so that all devices of a particular type can be managed as one.
+
+You will need to create appropriate hardware types in your system.
+
+Empty profiles can be assigned to a hardware type group either before or after registration.
+If an empty profile is not assigned to a hardware type group before registration, it will be assigned to group that best matches the product information available to it.
+
+For this procedure, you will require the system manufacturer name and product name for your terminal.
+
+.Procedure: Creating a Hardware Type group
+
+. Determine the hardware type group name.
+Prefix the name with [systemitem]``HWTYPE:``, followed by the system manufacturer name and product name, separated by a hyphen.
+For example:
++
+----
+HWTYPE:POSVendor-Terminal1
+----
++
+[IMPORTANT]
+====
+Only use colons (``:``), hyphens (``-``) or underscores (``_``) in hardware type group names.
+Spaces and other non-alphanumeric characters will be removed when the name is processed.
+====
++
+. In the {susemgr} {webui}, navigate to menu:Systems[System Groups], and click the btn:[Create Group] button.
+. In the [guimenu]``Create System Group`` dialog, create a new system group, using the hardware type group name you created earlier.
+
+
+== Assign and Configure the Saltboot formula for each Hardware Type Group
+
+TBD
+
+.Procedure: Configuring partitioning and assigned images
+
+. Open the details page for your new hardware type group, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``saltboot-formula`` and click btn:[Save].
+. Navigate to the new menu:Formulas[Saltboot] tab.
+. Configure the Saltboot formula according to the instructions in xref:retail:retail-formulas[].
+. Apply the highstate.
```

Comment:
Highstate can't be applied here, the newly created group is empty so there are no machines to apply highstate on.

The saltboot state is applied during terminal boot automatically.